acceptance/acceptance_test.go

```diff
@@ -150,6 +159,12 @@ func sharedSetup(tsEnv testScriptEnv) func(ts *testscript.Env) error {
 		ts.Setenv("ORG", tsEnv.org)
 		ts.Setenv("GH_TOKEN", tsEnv.token)
 
+		// Used for gh auth testing
+		ts.Setenv("GH_USER", tsEnv.user)
```

Comment:
**Food For Thought**

Don't need to get ahead of ourselves here but this looks like not _shared setup_, and maybe we need some other way to customise this for specific commands / testscripts.